Sources/Controllers/2.0/OAGPXListViewController.mm

```diff
@@ -594,11 +592,11 @@ - (void)onTrackRecordingChanged
     });
 }
 
-- (void)onGpxRouteCanceled
+- (void) onGpxRouteCanceled
 {
     dispatch_async(dispatch_get_main_queue(), ^{
         
-        if (_viewMode == kActiveTripsMode && _routeSectionIndex > -1)
+        if (_routeSectionIndex > -1)
```

Comment:
_routeSectionIndex is used here but never assigned?

Reply:
Maybe it would be a great idea not to use constants for sections at all since we will have a dynamic number of folders in the future.